### PR TITLE
Minutes tech lead 30 September 2022

### PR DESCRIPTION
**Pull request by valeriupredoi**
:beer:

### CODE REVIEW

File: Tech Lead/Minutes/20220930.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Tech Lead Call 30 Sept 2022
+
+**Attendance:** @zklaus @remi-kazeroni @sloosvel @bouweandela @schlunma @valeriupredoi
+
+- Bouwe: iris collaboration -> any more opinions
+  - Saskia: ESMF devs also part of iris? Yes,
```
Comment (bouweandela) [inline Tech Lead/Minutes/20220930.md:6]:
```suggestion
  - Saskia: [iris-esmf-regrid](https://github.com/SciTools-incubator/iris-esmf-regrid) devs also part of iris? Yes,
```

File: Tech Lead/Minutes/20220930.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - Monday 17th Oct get-together with iris folk
+  - Bouwe: priorities list - prob best to have it done before the meeting
+    -> can talk about it now, items from this meeting:
+    *NOTE from Klaus: look for "Feature: ESMValTool" label at iris GH*
+    - **IMPORTANT: saving multiple files in parallel with iris** - this will help a lot with (draft) PR https://github.com/ESMValGroup/ESMValCore/pull/1714
+      Klaus: `scheduler = dask.distributed` offers better performance, plus not `threading`; see also https://github.com/SciTools/iris/issues/4190 https://github.com/SciTools/iris/issues/3333
+    - Manu: https://github.com/SciTools/iris/issues/4860 (ugrid stuff)
+    - Saskia: iris is not recognizing one of CORDEX coords: https://github.com/SciTools/iris/issues/4830
+  - ESMF regridder users parallel session at the workshop
```
Comment (bouweandela) [inline Tech Lead/Minutes/20220930.md:15]:
```suggestion
  - iris-esmf-regrid regridder users parallel session at the workshop
```

**Review by bouweandela — COMMENTED** [review]
Thanks for organizing the meeting @valeriupredoi!

**Comment by valeriupredoi** [comment]
cheers @bouweandela :beer:

**Review by bouweandela — APPROVED** [review]

**Pull request merged**